### DISCUSSION TITLE
Reason Why the Latest Source Code Does Not Support the Electron Desktop Build

### DISCUSSION THREAD

**Discussion by jrt324** [foxglove/community#890] [Q&A]
I noticed that previous versions supported the Electron desktop build, but it seems to have been removed in the latest version. I’m curious about the considerations behind this decision. Could you kindly discuss?

previous versions of package.json
``` json
    "desktop:build:dev": "webpack --mode development --progress --config desktop/webpack.config.ts",
    "desktop:build:prod": "webpack --mode production --progress --config desktop/webpack.config.ts",
    "desktop:serve": "webpack serve --mode development --progress --config desktop/webpack.config.ts",
    "desktop:serve:quicklook": "webpack serve --mode development --progress --config desktop/webpack.quicklook.config.ts",
    "desktop:start": "electron desktop/.webpack"
```

And now it been Removed

**Comment by amacneil** [comment]
Hi @jrt324,

Correct, we are no longer maintaining the open source electron code as of https://github.com/foxglove/studio/pull/6895. The official [Foxglove desktop](https://foxglove.dev/download) adds extra commercial features for managing layouts, data, and extensions.

We removed the electron code from the studio repo, which is not used by our internal release process. Publishing a desktop app is complex, requires a signing certificate on macOS and Windows, and is not something we provide support for. Community members are welcome to maintain their own desktop derivatives, but we encourage use of our [official desktop binaries](https://github.com/foxglove/studio/releases) or our [open source web builds](https://github.com/orgs/foxglove/packages/container/package/studio) going forward.
